plugins/fireeye_hx/icon_fireeye_hx/util/endpoint.py

```diff
@@ -0,0 +1,14 @@
+def alerts() -> str:
+    return "alerts"
+
+
+def host_containment(agent_id: str) -> str:
+    return f"hosts/{agent_id}/containment"
+
+
+def hosts() -> str:
+    return "hosts"
+
+
+def version() -> str:
+    return "version"
```

Comment:
Wouldn't be better to create a class called `Endpoint` and to put all constant endpoints with formatting? 😃 
```suggestion
class Endpoint:
    ALERTS = "alerts"
    HOSTS = "hosts"
    HOST_CONTAINMENT = "hosts/{}/containment"
```